pkg/skaffold/util/term.go

```diff
@@ -35,3 +41,21 @@ func IsTerminal(w io.Writer) (uintptr, bool) {
 
 	return 0, false
 }
+
+func SupportsColor() (bool, error) {
+	res, err := colors()
+	if err != nil {
+		return false, fmt.Errorf("checking terminal colors: %w", err)
+	}
+
+	numColors, err := strconv.Atoi(strings.TrimSpace(string(res)))
+	if err != nil {
+		return false, err
+	}
+
+	return numColors != 0, nil
+}
+
+func tputColors() ([]byte, error) {
+	return exec.Command("tput", "colors").Output()
```

Comment:
This won't work for Windows, which normally does support ANSI sequences.

Reply:
realized this when testing locally 😅 I can add a case to this function to check OS